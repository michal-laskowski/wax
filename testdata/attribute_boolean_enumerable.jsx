export function attribute_boolean_enumerable() {
    return (<>
        <attr-known>
            <sample-01 draggable={true} />
            <sample-02 draggable={false} />
            <sample-03 spellcheck={true} />
            <sample-04 spellcheck={false} />
            <sample-05 draggable={"true"} />
            <sample-06 draggable={"false"} />
            <sample-07 spellcheck={"true"} />
            <sample-08 spellcheck={"false"} />
        </attr-known>
        <attr-aria>
            <sample-01 aria-checked={true} />
            <sample-02 aria-checked={false} />
            <sample-03 aria-selected={true} />
            <sample-04 aria-selected={false} />
        </attr-aria>
    </>)
}