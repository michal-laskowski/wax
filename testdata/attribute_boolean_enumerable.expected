
<attr-known>
    <sample-01 draggable="true"></sample-01>
    <sample-02 draggable="false"></sample-02>
    <sample-03 spellcheck="true"></sample-03>
    <sample-04 spellcheck="false"></sample-04>
    <sample-05 draggable="true"></sample-05>
    <sample-06 draggable="false"></sample-06>
    <sample-07 spellcheck="true"></sample-07>
    <sample-08 spellcheck="false"></sample-08>
</attr-known>

<attr-aria>
    <sample-01 aria-checked="true"></sample-01>
    <sample-02 aria-checked="false"></sample-02>
    <sample-03 aria-selected="true"></sample-03>
    <sample-04 aria-selected="false"></sample-04>
</attr-aria>